go/engine/identify2_with_uid.go

```diff
@@ -347,6 +347,11 @@ func (e *Identify2WithUID) run(m libkb.MetaContext) {
 	err := e.runReturnError(m)
 	e.unblock(m /* isFinal */, true, err)
 
+	if !e.arg.IdentifyBehavior.IsChatIdentify() {
```

Comment:
Need to be careful not to cause an infinite loop here.

Alternate solution is to export the results of this identify to TLFBreaks and call ` globals.CtxIdentifyNotifier(ctx)` directly from a new notification handler, without running an identify. But I thought that might end up out of sync with the chat's regular identifies which runs Identify2 with different parameters than the original call may be using.

Reply:
Maybe safer would be to only do this on the GUI profile behavior.

Reply:
We can always just not use `UserChanged` here